### DISCUSSION TITLE
DRF version 3.15.1 creates strange api-root routes

### DISCUSSION THREAD

**Discussion by lukklim** [encode/django-rest-framework#9419] [Potential Issue]
Hello.
I have some tests to make sure my url routes are properly set up and noticed that new version of DRF 3.15.1 creates url paths for api-root strangely(in my opinion it is wrong). It is not a major problem, because everything else seems working just fine, but worth noting.

DRF 3.14.0 example:
<img width="504" alt="Screenshot 2024-05-30 at 11 14 32" src="https://github.com/encode/django-rest-framework/assets/131126623/2019b333-26fd-468f-8fdf-5d5ba7650427">

DRF 3.15.1 example:
<img width="593" alt="Screenshot 2024-05-30 at 11 12 38" src="https://github.com/encode/django-rest-framework/assets/131126623/9d8bc6bd-e4aa-4356-9f70-0681ebefb98c">

Examples were created by adding code below to users app:
```
router = DefaultRouter()

urlpatterns = [
    path("", include(router.urls)),
]
```
Tested with `Django==5.0.1` and `Django==5.0.6`

**Comment by auvipy** [comment]
I think it is expected behavior. because DRF is now able to use path in it's actions.
